docs/userguide/modules/getting-started/partials/connect-petclinic-pgcluster.adoc

```diff
@@ -0,0 +1,91 @@
+== Connect the application to the database
+
+Suppose the {servicebinding-title} is not present. In that case, the
+application\'s admin needs to extract all the configuration details and
+create a Secret resource and expose it to the application through volume
+mount in Kubernetes. The steps would be something like this:
+
+. Identify the required values for connecting the application to the
+database
+. Locate the resources where the values are present
+. Take the values from different resources and create a Secret resource
+. Mount the Secret resource into the application
+. Depending on the application requirement the values should be exposed
+as env var or file.
+
+In this quick start, we are going to leverage {servicebinding-title} as a way
+to easily and safely connect the application to the database service. In
+order to do that, we\'ll need to create a `ServiceBinding` resource which
+will trigger the {servicebinding-title} to project the binding data
+into the application.
+
+=== Create the `ServiceBinding` resource to project the binding data:
+
+The `.spec` has two sections. The first is a list of service
+resources (`.spec.services`), and the second one is the application
+(`.spec.application`). The services resources points to the database\'s
+service resources. How the values are exposed from service resources are
+explained xref:exposing-binding-data:intro-expose-binding.adoc[Exposing Binding Data]
+section. The application points to a `Deployment`
+or any resource that is PodSpec compatible.
+
++++ <details><summary> +++
+`ServiceBinding` resource to project the binding data 
++++ </summary><div> +++
```

Comment:
I think you can use here asciidoc caption block instead:

```
.`ServiceBinding` resource to project the binding data 
[source,yaml]
```

If yes, please replace this across all appearances.

Reply:
I'm using this approach to have the source code of the actual YAML hidden/collapsed in the rendered page and only to be shown when user is interested in seeing it by clicking on the title. I like that approach much better then simply including the whole YAMLs to appear in the document out-of-the-box making the document large.

IMHO, it makes it easier for a reader to read the document.

Reply:
ok